packages/hydrogen-react/src/CartProvider.doc.ts

```diff
@@ -12,7 +12,7 @@ const data: ReferenceEntityTemplateSchema = {
     },
   ],
   description: `
-    The \`CartProvider\` component synchronizes the state of the Storefront API Cart and a customer's cart, and allows you to more easily manipulate the cart by adding, removing, and updating it. It could be placed at the root of your app so that your whole app is able to use the \`useCart()\` hook anywhere.\n\nThere are props that trigger when a call to the Storefront API is made, such as \`onLineAdd={}\` when a line is added to the cart. There are also props that trigger when a call to the Storefront API is completed, such as \`onLineAddComplete={}\` when the fetch request for adding a line to the cart completes.
+    The \`CartProvider\` component synchronizes the state of the Storefront API Cart and a customer's cart, and allows you to more easily manipulate the cart by adding, removing, and updating it. It could be placed at the root of your app so that your whole app is able to use the \`useCart()\` hook anywhere.\n\nThere are props that trigger when a call to the Storefront API is made, such as \`onLineAdd={}\` when a line is added to the cart. There are also props that trigger when a call to the Storefront API is completed, such as \`onLineAddComplete={}\` when the fetch request for adding a line to the cart completes.\n\nThe \`CartProvider\` component must be a descendant of the \`ShopifyProvider\` component.
```

Comment:
```suggestion
    The \`CartProvider\` component synchronizes the state of the Storefront API Cart and a customer's cart, and allows you to more easily manipulate the cart by adding, removing, and updating it. It could be placed at the root of your app so that your whole app is able to use the \`useCart()\` hook anywhere.\n\nThere are props that trigger when a call to the Storefront API is made, such as \`onLineAdd={}\` when a line is added to the cart. There are also props that trigger when a call to the Storefront API is completed, such as \`onLineAddComplete={}\` when the fetch request for adding a line to the cart completes.\n\nThe \`CartProvider\` component must be a descendant of the \`ShopifyProvider\` component\n.
```